### PR TITLE
summary added for Metadata Subgroup Call D

### PR DESCRIPTION
**Pull request by ericaconnell**

### CODE REVIEW

**Review by jandrieu — APPROVED** [review]
LGTM

**Comment by jandrieu** [comment]
IMO, someone other than Legendary Requirements personnel should review and merge these notes. @okwme @ig-shaun @deepanshutr

**Comment by ericaconnell** [comment]
I fixed the merge. It should be ready for review.

File: markdowns/workingGroup/meetings.md [hunk 1]
```diff
@@ -352,6 +352,119 @@ Slides available at: https://docs.google.com/presentation/d/1mMPIVfjFBOh9qHL5aW4
 
 **10th December 2020**
 
+
+SUMMARY:
+<ol type = "1">
```
Comment (deepanshutr) [inline markdowns/workingGroup/meetings.md:357]: Requesting the HTML  tags to be removed and be replaced with the text in markdown format. The rest of the doc has been migrated to the same, can be used as a reference.

**Comment by jandrieu** [comment]
@deepanshutr The formatting of the earlier calls was broken because when the markdown gets translated into html, we lost the outline formatting. The html approach fixes that.

**Comment by deepanshutr** [comment]
@jandrieu as mentioned in the closing comments on the issue https://github.com/interNFT/documentation/issues/22 , the build failure occurred because some of the list HTML tags were not closed. This might have been missed in the review because HTML is a little arcane to review as text. Markdown is a lot easier to read and review, hence have removed HTML formatting from all other markdowns as well. Requesting this PR to adhere to markdown formatting too.

**Comment by jandrieu** [comment]
@deepanshutr We can fix html problems. However, the garbling of markdown to html, as you can see on the first call record, is out of my control without using HTML. Markdown lists do not translate to HTML with this build system and pre-formatted lists leverages neither the benefits of markdown nor html.

For now, I believe the best option for good formatting is using html as you see in the calls we've handled.

If you'd like to take on the task of correcting and maintaining the summary, we'd be happy to hand that over.

@ig-shaun Do you have a preference?

**Comment by deepanshutr** [comment]
@jandrieu the formatting for the calls you've handled has been changed from HTML to markdown too, hope there are no gaps in the formatting. Will be happy to take up the maintenance of the text.

**Review by deepanshutr — APPROVED** [review]
Reviewed, build passing, ready to merge.

**Comment by jandrieu** [comment]
Ok. We can use that pattern. We'll clean up the old ones with the markdown that fails to survive to html.

**Pull request merged**